spec/interface_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'interface'
+
+describe Interface do
+
+  it "Is able to call on the takeaway menu." do
+    expect(subject.food).to receive(:menu)
+    subject.food_menu
+  end
+
+  it "Is able to display the options_menu." do
+    options = "------------------------------------------\n1. Place your order\n2. View your order\n9. Exit\n"
+    expect { subject.options_menu }.to output(options).to_stdout
+  end
+
+  it "Is able to accept user input '1' and call on :place_order." do
```

Comment:
I would double check how this is being called. As you interface does not call an PalaceOrder class and place_order is a method in that class this should in theory fail without some kind of double... We can maybe go through together and find an appropriate  double as I am slightly unsure too.